dev/com.ibm.ws.microprofile.metrics_fat/fat/src/com/ibm/ws/microprofile/metrics/tck/launcher/MetricsAuthenticationTest.java

```diff
@@ -79,7 +79,13 @@ private void addFeature(String feature) throws Exception {
     }
 
     private void waitForMetricsEndpoint(LibertyServer server) throws Exception {
-        assertNotNull("Web application is not available at */metrics/", server.waitForStringInLog("CWWKT0016I.*/metrics/", TIMEOUT * 2, server.getDefaultLogFile()));
+        assertNotNull("Web application is not available at */metrics/", server.waitForStringInLogUsingMark("CWWKT0016I.*/metrics/", server.getDefaultLogFile()));
```

Comment:
Do we need to specify `server.getDefaultLogFile()` here as messages.log is the default log file.

Reply:
I did that just to be 100% sure that it will wait for the string in message.log and not some other log

Reply:
Nevermind. waitForStringInLogUsingMark will use message.log by default so i will remove server.getDefaultLogFile() from all instances of waitForStringInLogUsingMark